docs/src/app/components/pages/home.jsx

```diff
@@ -8,6 +8,7 @@ const { StylePropable, StyleResizable } = Mixins;
 const { Colors, Spacing, Typography } = Styles;
 const ThemeManager = Styles.ThemeManager;
 const DefaultRawTheme = Styles.LightRawTheme;
+require('github-markdown-css/github-markdown.css');
```

Comment:
You can do it this way if you have a webpack loader for style configured, like I did it in `docs/webpack-dev-server.config.js`and `docs/webpack-production-server.config.js`. Might be that you must `npm install` first.
Anyway this is purely academic, because I'm going to remove this stuff as I want to stick with the existing documentation standard. (you're last comment....)